backup/src/test/java/io/camunda/zeebe/backup/management/BackupServiceImplTest.java

```diff
@@ -40,12 +43,21 @@ class BackupServiceImplTest {
   @Mock InProgressBackup inProgressBackup;
   @Mock BackupStore backupStore;
 
+  @Mock BackupStatus notExistingBackupStatus;
```

Comment:
:wrench: this looks like it is unused.

Reply:
This is required because all takeBackup operation first check the status of existing backup.